consensus/hotstuff/verification/staking_signer.go

```diff
@@ -23,16 +23,14 @@ type StakingSigner struct {
 
 // NewStakingSigner creates a new combined signer with the given dependencies:
 // - the staking signer is used to create and verify aggregatable signatures for Hotstuff
```

Comment:
I think this could be updated:
```suggestion
// NewStakingSigner instantiates a StakingSigner, which signs votes and 
// proposals with the staking key.  The generated signatures are aggregatable.
```